controller/src/api/external/routes/mod.rs

```diff
@@ -17,7 +17,21 @@ type Handler = fn(
     &route_recognizer::Params,
     &Connection,
     &Sender<ApiChannel>,
-) -> Result<tiny_http::Response<io::Cursor<Vec<u8>>>, api::RikError>;
+) -> Result<tiny_http::Response<io::Cursor<Vec<u8>>>, anyhow::Error>;
+
+type HttpResult<T = io::Cursor<Vec<u8>>> = Result<Response<T>, anyhow::Error>;
+
+pub enum ContentType {
+    JSON,
+}
+
```

Comment:
You could even make things simpler, by making a `Into<tiny_http::Header>` implementation as it's safe to unwrap

Reply:
good idea, but i think `Into<tiny_http::HeaderField>` is better, what do you think ?

Reply:
Good idea